tools/experimental/ltscombine/lts_combine.cpp

```diff
@@ -128,9 +128,9 @@ bool is_allowed(const std::vector<core::identifier_string_list>& allowed, const
     return true;
   }
 
-  for (const auto& it : allowed)
+  for (const auto& action : allowed)
```

Comment:
Deze auto zou door het type moeten worden vervangen. Wat is het type, een multi-actie, of een actie label?? Of een multi-actie-naam?